tests/schemas/test_ticket_schema.py

```diff
@@ -4,7 +4,7 @@
 
 @pytest.mark.usefixtures("empty_test_db")
 class TestTicketValidations:
-    def test_valid_payload(self, create_tenant, create_admin_user):
+    def test_valid_payload_without_note(self, create_tenant, create_admin_user):
```

Comment:
I don't think this method name needs to be renamed. For me `test_valid_payload` tests the minimum set of required attributes. A Ticket accepts many more attributes then simply a tenant_id and an author_id, but these are apparently the only attributes that are required for the payload to be valid. So let's not rename this, otherwise we could end up with a very long name if we're going to state every attribute that is not part of the valid attribute in the method name.